static/js/github-edit-button.js

```diff
@@ -0,0 +1,32 @@
+(function addGithubEditButton() {
+    $('#edit-on-github').click(function (e) {
+        if (location.hostname.indexOf("localhost") != -1 ) 
+        {
+            alert("Disabled for localhost")
+            return;
+        }
+        var path = location.pathname;
+        if (path.indexOf("/", path.length - 1) !== -1) path = path.slice(0, -1);
```

Comment:
so this line just trims a "/" suffix if it exists..
we can make it more readable with something like:
```
path = path.replace(/\/$/, '');
```